src/formatter/parallel.rs

```diff
@@ -26,75 +26,66 @@ where
         }
     }
 
-    fn replay_suite(&self, suite: &SuiteHeader, report: &SuiteReport) -> io::Result<()> {
-        self.serial.enter_suite(suite)?;
-        self.replay_context(None, report.get_context())?;
-        self.serial.exit_suite(suite, report)?;
-
-        Ok(())
+    fn replay_suite(&self, suite: &SuiteHeader, report: &SuiteReport) {
```

Comment:
I need to think about this...

Reply:
It's the cleanest solution I could come up with, that is guaranteed to generate same output for both serial and parallel formatters, yet manages to share the logic.

For the parallel formatter I'm thinking about displaying a progress bar via the **indicatif** crate, while the tests run. Replacing them with the actual (replayed) logs once the suite exits.

Reply:
yes, it's a good idea. I was thinking about making the replay_* methods part of a public trait with empty default impl. Even sequential reporters could beneficiate from it (like a summary of the failures at the end).

Also, big thumb up 👍 . I really like the look of it !